packages/components/src/index.ts

```diff
@@ -62,7 +62,7 @@ export {
 	CompositeItem as __unstableCompositeItem,
 	useCompositeState as __unstableUseCompositeState,
 } from './composite/legacy';
-export { Composite } from './composite';
+export { Composite, useCompositeStore } from './composite';
```

Comment:
I had actually forgotten to export `useCompositeStore` in previous PRs 😅

Reply:
Good timing to find out about it 😅